Gateway/Command/Initialize.php

```
@@ -0,0 +1,84 @@
+{
+    /**
+     * @param SubjectReader $subjectReader
+     * @param CommandManagerPoolInterface $commandManagerPool
+     * @param LoggerInterface $logger
+     */
+    public function __construct(
+        private readonly SubjectReader $subjectReader,
+        private readonly CommandManagerPoolInterface $commandManagerPool,
+        private readonly LoggerInterface $logger
+    ) {
+    }
+
+    /**
+     * Execute function
+     *
+     * @param array $commandSubject
+     *
+     * @return $this
+     * @throws LocalizedException
+     */
+    public function execute(array $commandSubject): static
+    {
+        /** @var PaymentDataObjectInterface $payment */
+        $paymentData = $this->subjectReader->readPayment($commandSubject);
+        $stateObject = $this->subjectReader->readStateObject($commandSubject);
+
+        /** @var InfoInterface $payment */
+        $payment = $paymentData->getPayment();
+        $payment->setIsTransactionPending(true);
+        $payment->setIsTransactionIsClosed(false);
+        $order = $payment->getOrder();
+        $order->setCanSendNewEmailFlag(false);
+
+        $stateObject->setIsNotified(false);
+        $stateObject->setState(Order::STATE_NEW);
+        $stateObject->setStatus('pending');
+        $stateObject->setIsNotified(false);
+
+        $this->cratePayment($paymentData);
+
+        return $this;
+    }
+
+    /**
+     * Create payment in Nexi Gateway
+     *
+     * @param PaymentDataObjectInterface $payment
+     *
+     * @return ResultInterface|null
+     * @throws LocalizedException
+     */
+    public function cratePayment(PaymentDataObjectInterface $payment): ?ResultInterface
+    {
+        try {
+            $commandPool = $this->commandManagerPool->get(Config::CODE);
+            $result      = $commandPool->executeByCode(
+                commandCode: 'create_payment',
+                arguments  : ['payment' => $payment,]
+            );
+        } catch (\Exception $e) {
```

dont catch all Exceptions
and besides that why here Exception is `\Exception` and in other files its put into use statement
please configure cs fixer to have it consistent in all files, i do prefer to have global namespace classes with \ in code then in use statement